notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>&nbsp;A geodesic is the shortest distance</p><p>--&gt; the geodesic is the path of shortest distance (not the distance itself)</p><p>-&gt; When the manifold is equipped with a so-called "Riemannian metric" (which will be introduced in a later notebook), a geodesic is the path of shortest distance that...</p><p> </p><p>NIT: Figure (a), Figure (b) etc</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I changed the sentence to: When the manifold is equipped with a so-called "Riemannian metric" (which will be introduced in a later notebook), we can define geodesics on the manifold. The geodesic is the path of shortest distance that a particle can travel in the space that it is in.</p>